src/nft/components/explore/TrendingCollections.tsx

```diff
@@ -69,9 +72,25 @@ const StyledSelectorText = styled(ThemedText.SubHeader)<{ active: boolean }>`
   color: ${({ theme, active }) => (active ? theme.textPrimary : theme.textSecondary)};
 `
 
+function convertTimePeriodToHistoryDuration(timePeriod: TimePeriod): HistoryDuration {
```

Comment:
might be nicer just to do a mapping like this:

```
const TimePeriodToHistoryDuration = {
  [TimePeriod.OneDay]: HistoryDuration.Day,
  [TimePeriod.SevenDays]: HistoryDuration.Week,
  [TimePeriod.ThirtyDays]: HistoryDuration.Month,
  [TimePeriod.AllTime]: HistoryDuration.Max,
}
```

Reply:
This is a temporary helper fn as once the flag is merged in, TimePeriod will be deprecated in favor of the generated HistoryDuration enum